src/main/java/domain/CarDTO.java

```diff
@@ -0,0 +1,10 @@
+package domain;
+
+public class CarDTO {
```

Comment:
```suggestion
public class CarDto {
```

클래스명은 일반적으로 upper camel case로 표기해요!

Reply:
줄임말이라 저렇게 써도 괜찮다고 생각했는데, 덕분에 어디에나 일관성있게 upper camel case로 사용하는 이유를 이해할 수 있었습니다.

> Consider multiple such terms: compare getSQLDBIOXML() to getSqlDbIoXml(), https://stackoverflow.com/a/8277473

별 의미는 없고 찾아보다가 단번에 이해돼서 긁어왔습니다... 감사합니다.